src/components/Video.js

```diff
@@ -65,6 +65,7 @@ export default class Video extends React.Component<Props> {
         controls: true,
         // Try fetching length of video etc
         preload: 'metadata',
+        poster: images[0] && images[0].image ? images[0].image : '',
```

Comment:
can you change this to prefer `images[0].secure_url` if it's available. Probably best to do it in an `if` statement then, instead of inline.

Reply:
you sure it has a `secure_url`?
https://github.com/GetStream/react-activity-feed/blob/master/custom-definitions/getstream.js#L67

Reply:
It can have it, `og:image:secure_url` is a official attribute: http://ogp.me/

You can add it to the type definition file.